docs(all): Remove insensitive wordings from the docs (#3320)
webpack/webpack.js.org: src/content/guides/asset-management.md

</note>
```

> Now you can `import` any one of those four types of data (JSON, CSV, TSV, XML) and the `Data` variable you import it to will contain parsed JSON accessible in your code:

Reviewer: ```suggestion
Now you can `import` any one of those four types of data (JSON, CSV, TSV, XML) and the `Data` variable you import it to will contain parsed JSON for consumption:
```
